### PR TITLE
fix: Stableswap decimal error

### PR DESCRIPTION
**Pull request by chefjackson**

### CODE REVIEW

File: packages/smart-router/evm/stableSwap/getLPOutput.ts [hunk 1]
```diff
@@ -44,13 +36,13 @@ export function getLPOutput({
   for (const [i, balance] of balances.entries()) {
     const amount = amounts[i] || CurrencyAmount.fromRawAmount(balance.currency, 0)
     invariant(
-      amount.currency.wrapped.equals(balance.currency.wrapped),
+      amount.currency.equals(balance.currency),
```
Comment (chef-jojo) [inline packages/smart-router/evm/stableSwap/getLPOutput.ts:39]: need wrap for native currency passing in

File: packages/smart-router/evm/stableSwap/getSwapOutput.ts [hunk 2]
```diff
@@ -26,13 +27,13 @@ export function getSwapOutput({
   let j: number | null = null
   const balances: JSBI[] = []
   for (const [index, b] of balanceAmounts.entries()) {
-    balances.push(b.quotient)
-    if (b.currency.wrapped.equals(amount.currency.wrapped)) {
+    balances.push(getRawAmount(b))
+    if (b.currency.equals(amount.currency)) {
```
Comment (chef-jojo) [inline packages/smart-router/evm/stableSwap/getSwapOutput.ts:31]: ditto?

File: packages/smart-router/evm/stableSwap/getLPOutput.ts (hunk 1 above)
Comment (chefjackson) [reply packages/smart-router/evm/stableSwap/getLPOutput.ts]: done

File: packages/smart-router/evm/stableSwap/getSwapOutput.ts (hunk 2 above)
Comment (chefjackson) [reply packages/smart-router/evm/stableSwap/getSwapOutput.ts]: done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**